parm/config/gfs/config.atmanl

```diff
@@ -6,7 +6,8 @@
 echo "BEGIN: config.atmanl"
 
 export OBS_LIST="${PARMgfs}/gdas/atm/obs/lists/gdas_prototype_3d.yaml.j2"
-export JEDIYAML="${PARMgfs}/gdas/atm/variational/3dvar_drpcg.yaml.j2"
+export VARYAML="${PARMgfs}/gdas/atm/variational/3dvar_drpcg.yaml.j2"
```

Comment:
Renaming  to `VARYAML` and the associated changes will most certainly break the snow and aerosol tests.

`JEDIYAML` and `JEDIEXE` are required for all analysis tasks.
The issue we now face is that we are calling multiple executables (and their associated yamls) while using the same `analysis` class.

Reply:
C96C48_ufs_hybatmDA CI tests on Orion are failing in enkfgdasatmensanlinit.  The traceback suggests the failure is related to this change and related changes in `analysis.py` and/or `atm_analysis.py`.  Not sure yet if the solution is to modify `atmens_analysis.py` and `config.atmensanl`, modify some changes in this PR, a combination of both, or something else.